azure-storage/spring-cloud-starter-stream-sink-wasb/src/main/java/org/springframework/cloud/stream/app/azure/storage/sink/WasbSinkConfiguration.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.app.azure.storage.sink;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.cloud.stream.annotation.EnableBinding;
+import org.springframework.cloud.stream.messaging.Sink;
+import org.springframework.context.annotation.DependsOn;
+import org.springframework.integration.annotation.ServiceActivator;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.MessagingException;
+
+// Include the following imports to use blob APIs.
+import com.microsoft.azure.storage.CloudStorageAccount;
+import com.microsoft.azure.storage.blob.BlobContainerPermissions;
+import com.microsoft.azure.storage.blob.BlobContainerPublicAccessType;
+import com.microsoft.azure.storage.blob.CloudBlob;
+import com.microsoft.azure.storage.blob.CloudAppendBlob;
+import com.microsoft.azure.storage.blob.CloudBlobClient;
+import com.microsoft.azure.storage.blob.CloudBlobContainer;
+import com.microsoft.azure.storage.blob.CloudBlockBlob;
+
+
+/**
+ * @author Kyle Dunn
+ */
+@EnableBinding(Sink.class)
+@EnableConfigurationProperties(WasbSinkProperties.class)
+@SpringBootApplication
+public class WasbSinkConfiguration {
+
+    protected static Logger LOG = LoggerFactory.getLogger(WasbSinkConfiguration.class);
+
+    @Autowired
+    private WasbSinkProperties properties;
+
+    private CloudBlob blobService;
+
+    @Autowired
+    @DependsOn("WasbSinkProperties")
```

Comment:
Most likely not - probably a troubleshooting artifact. I'll remove it and rebuild to double check.